backend/src/tasks/test/scheduler.test.ts

```diff
@@ -211,6 +214,72 @@ describe('scheduler', () => {
 
       expect(runCommand).toHaveBeenCalledTimes(0);
     });
+    test('should not run a scan when scan is a SingleScan and has finished recently.', async () => {
+      const scan = await Scan.create({
+        name: 'findomain',
+        arguments: {},
+        frequency: 999,
+        isSingleScan: true,
+        lastRun: new Date()
+      }).save();
+      const organization = await Organization.create({
+        name: 'test-' + Math.random(),
+        rootDomains: ['test-' + Math.random()],
+        ipBlocks: [],
+        isPassive: false
+      }).save();
+      await ScanTask.create({
+        organization,
+        scan,
+        type: 'fargate',
+        status: 'finished',
+        finishedAt: new Date()
+      }).save();
+
+      await scheduler(
+        {
+          scanId: scan.id,
+          organizationId: organization.id
+        },
+        {} as any,
+        () => void 0
+      );
+
+      expect(runCommand).toHaveBeenCalledTimes(0);
+    });
+    test('should not run a scan when scan is a SingleScan and after sufficient time has passed', async () => {
```

Comment:
Isn't this test the same as the earlier test?

Reply:
Yes, changed to one test